aea/aea_builder.py

```diff
@@ -441,7 +442,10 @@ def set_decision_maker_handler(
         dotted_path, class_name = decision_maker_handler_dotted_path.split(
             DOTTED_PATH_MODULE_ELEMENT_SEPARATOR
         )
-        module = load_module(dotted_path, file_path)
+        if file_path is None:
+            module = import_module(dotted_path)
+        else:
+            module = load_module(dotted_path, file_path)
```

Comment:
Need to add try/except

Reply:
I think somewhere we need to explain that `file_path` overwrites the behaviour of `dotted_path`

Reply:
It does not overwrite..